docs/source/index.rst

```diff
@@ -1,33 +1,10 @@
 Snowpark API Reference (Python)
 ===========================================================
 
-Snowpark is a new developer experience that provides an intuitive API for querying and handling data.
-Snowpark simplifies the process of building complex data pipelines and allows you to interact with
-Snowflake directly without moving data to the system where your application code runs. For more
-information, see the `Snowpark Developer Guide for Python <https://docs.snowflake.com/en/developer-guide/snowpark/python/index.html>`_.
-
 .. toctree::
-   :maxdepth: 2
+   :maxdepth: 1
 
-   session
-   io
-   dataframe
-   column
-   types
-   row
-   functions 
-   window 
-   grouping 
-   table_function
-   table
-   async_job
-   stored_procedures
-   udf
-   udaf
-   udtf
-   observability
-   files
-   context
-   exceptions
+   snowpark/session
```

Comment:
Why do we need to put session here?

Reply:
This is based on the outline Sri wanted: https://snowflakecomputing.atlassian.net/browse/SNOW-1320543
refer to google doc comment in the above link ^

Essentially the landing page should be
- Session
- Snowpark Python API
- Snowpark pandas API